source/ADIOS.cpp

```diff
@@ -25,10 +25,8 @@
 #include "engine/bp/BPFileReader.h"
 #include "engine/bp/BPFileWriter.h"
 
-#ifdef ADIOS_HAVE_DATAMAN // external dependencies
 #include "engine/dataman/DataManReader.h"
```

Comment:
Yep.  Keep the ifdefs.  Regarding workflow, just keep using this pr and just add more commits to the existing branch.  The or tracks your branch so when additional commits are added to the branch then the or is automatically updated.